docs/management/advanced-options.asciidoc

```diff
@@ -22,6 +22,8 @@ compatible with other configuration settings. Deleting a custom setting removes
 .Kibana Settings Reference
 [horizontal]
 `query:queryString:options`:: Options for the Lucene query string parser.
+`search:queryLanguage`:: Query language used by the query bar. Optionally enable Kuery, an experimental new language built specifically for Kibana.
```

Comment:
How about listing the valid choices here and making it clear what the default is.